test/setup.js

```diff
@@ -13,3 +13,4 @@ Object.keys(document.defaultView).forEach((property) => {
 global.navigator = {
   userAgent: 'node.js'
 };
+global.document.documentElement.dataset = {};
```

Comment:
This was added because the `DragReordable` component will error when Enzyme uses `mount` instead of shallow, because the component searches for `document.documentElement.dataset`.